indico/modules/designer/client/js/index.js

```diff
@@ -746,6 +757,9 @@
 
     backsideTemplateID = backsideTemplate.id;
 
+    // Get image placeholder list
+    imageTypes = image_types;
```

Comment:
Oh yuck, I was just about to comment on why not just put `imageTypes` in the function's argument list, then I realized the huge mess we have there with global-like variables...

I don't think this line needs a comment btw.